source/index.html.md.erb

```diff
@@ -37,7 +37,7 @@ Standards are based on formal decisions made by the tech forum. The standards ha
 
 The following standards will help you name your service, choose an appropriate programming language and where to store your code:
 
-- [naming things](standards/naming-things.html)
+- [how to name things](standards/naming-things.html)
```

Comment:
This is pedantry and I don't really care that much, but if we're changing things to try and get a consistent style this doesn't seems to match the other things in this list…

To me, it'd make sense to be either:

- how to name things
- how to choose programming languages
- how to store source code

or

- naming things
- choosing programming languages
- storing source code

:man_shrugging:

Reply:
I agree, I'm working my way through the published guidance and will update as I go through each piece 😄